Makefile.dep

```diff
@@ -8,6 +8,10 @@ OLD_USEPKG := $(sort $(USEPKG))
 # pull dependencies from drivers
 include $(RIOTBASE)/drivers/Makefile.dep
 
+ifneq (,$(filter pm_custom,$(USEMODULE)))
+  USEMODULE := $(filter-out pm_layered,$(USEMODULE))
```

Comment:
I think it makes more sense to error out and tell the user that they need to choose one or the other. If any drivers depend on pm_layered when they shouldn't, they should be fixed.

Reply:
I did it this way because pm_layered gets selected by default in the board Makefile or Makefile.dep, thus if not wanted, the module has to be filtered out here.